AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -79,53 +80,65 @@ The Alembic exporter can be customized by setting the properties on this compone
 
 | Property:| Function: |
 |:---|:---| 
-|__Output Path__ |Specify the location where the Alembic Exporter will save the ABC file. |
+|__Output Path__ |Specify the location where the Alembic Exporter will save the ABC file. By default the output path is relative to the current Unity project path. |
 |__Archive Type__ |Choose the Alembic format specification, the default is Ogawa and provides smaller and better performance than HDF5.|
```

Comment:
...the default is Ogawa which produces smaller files and better performance than HDF5.